Source/WebCore/platform/graphics/skia/ImageBufferSkiaAcceleratedBackend.h

```diff
@@ -41,6 +41,7 @@ class ImageBufferSkiaAcceleratedBackend final : public ImageBufferSkiaSurfaceBac
     WTF_MAKE_NONCOPYABLE(ImageBufferSkiaAcceleratedBackend);
 public:
     static std::unique_ptr<ImageBufferSkiaAcceleratedBackend> create(const Parameters&, const ImageBufferCreationContext&);
+    static std::unique_ptr<ImageBufferSkiaAcceleratedBackend> create(const Parameters&, const ImageBufferCreationContext&, sk_sp<SkSurface>&&);
```

Comment:
If you keep this create, I guess it could be private.

Reply:
Good point, will fix and then land! Thanks @carlosgcampos.